src/topp/IDFileConverter.cpp

```diff
@@ -329,6 +330,20 @@ class TOPPIDFileConverter :
       else if (in_type == FileTypes::IDXML)
       {
         IdXMLFile().load(in, protein_identifications, peptide_identifications);
+        // get spectrum_references from the mz data, if necessary:
+        if (!mz_file.empty())
```

Comment:
The case here is settled, yes, but we still had a discussion about `SpectrumMetaDataLookup::readMzFileMetaData` which wasn't fully resolved. See: https://github.com/OpenMS/OpenMS/pull/2498#discussion_r111946179